ml_mr/estimation/quantile_iv.py

```diff
@@ -290,6 +292,171 @@ def from_results(cls, dir_name: str) -> "QuantileIVEstimator":
         return cls(exposure_network, outcome_network, meta=meta, covars=covars)
 
 
+class QuantileIVLinearEstimator(MREstimatorWithUncertainty):
+    """
+    QuantileIV estimator that uses the linear inference strategy, which
+    consists of performing IV linear regression manually in the new
+    representation of the exposure learned by the outcome neural network.
+    """
+    def __init__(
+        self,
+        dataset: IVDataset,
+        parent: QuantileIVEstimator
+    ):
+        self.parent = parent
+        self.exposure_network = parent.exposure_network
+        self.outcome_network = parent.outcome_network
+        self.covars = parent.covars
+        self.pca = self.init_PCA()
+        self.dataset = dataset
+        self.h = self.linear_inference(dataset)
+
+    def iv_reg_function(
+        self,
+        x: torch.Tensor,
+        covars: Optional[torch.Tensor] = None,
+        alpha: float = 0.1
```

Comment:
I will transition the default to 0.05 everywhere when I have time. I suggest you apply it here already.

```suggestion
        alpha: float = 0.05
```